src/services/userService.ts

```diff
@@ -0,0 +1,45 @@
+import usersDao from '../models/userDao';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+
+const jwtSecret = process.env.SECRET_KEY;
+
+const signUp = async (
+  nickname: string,
+  password: string,
+  email: string,
+  sort_id: number,
+  is_admin: number
+) => {
+  const salt = await bcrypt.genSalt();
+  const hashedPw = bcrypt.hashSync(password, salt);
+  await usersDao.signUp(nickname, hashedPw, email, sort_id, is_admin);
+};
+
+const logIn = async (email: string, password: string) => {
+  const userInfo = await usersDao.logIn(email);
+  if (!userInfo) {
+    const error = new Error("User doesn't exist");
```

Comment:
```suggestion
    const error = new Error("User_doesn't_exist");
```
에러메세지의 빈칸은 언더바로 구분해주세요